app/models.py

```diff
@@ -1462,10 +1464,12 @@ def validate(self, enforce_required=True, required_fields=None, max_day_rate=Non
 
     def serialize(self):
         data = self.data.copy()
+        parent_brief = self.brief.serialize()
+        parent_brief_fields = ['id', 'title', 'status', 'applicationsClosedAt']
```

Comment:
Why do we restrict ourselves to these fields? We are already running `brief.serialize` above it and so there will be no further expense to add everything? (Not that saying, yes let's add everything but just wondering if there is a particular reason for it being good to not add more).

Also we won't run in any problems with it for this case, but if we at a later point added brief responses to the `brief.serialize` method then we would run into an infinite loop. Kind of links in to a point that I think George made about the ability to serialize an object without including any of it's related data objects so we can make sure serialisation doesn't propagate out too many other serialises.